modopt/math/stats.py

```diff
@@ -60,22 +62,22 @@ def gaussian_kernel(data_shape, sigma, norm='max'):
 
     """
     if not import_astropy:  # pragma: no cover
-        raise ImportError('Astropy package not found.')
+        raise ImportError("Astropy package not found.")
 
-    if norm not in {'max', 'sum', 'none'}:
+    if norm not in {"max", "sum"}:
         raise ValueError('Invalid norm, options are "max", "sum" or "none".')
 
     kernel = np.array(
         Gaussian2DKernel(sigma, x_size=data_shape[1], y_size=data_shape[0]),
     )
 
-    if norm == 'max':
+    if norm == "max":
         return kernel / np.max(kernel)
 
-    elif norm == 'sum':
+    elif version.parse(astropy_version) < version.parse("5.2"):
         return kernel / np.sum(kernel)
 
-    elif norm == 'none':
+    else:
         return kernel
```

Comment:
The documentation should be updated to reflect this new behavior

Reply:
Yep, thanks @paquiteau, sorted!